samples/Azure.AI.DocumentTranslation/Generated/DocumentTranslationClient.cs

```diff
@@ -127,43 +119,35 @@ public virtual async Task<Response> GetDocumentStatusAsync(Guid id, Guid documen
         /// <param name="documentId"> Format - uuid.  The document id. </param>
         /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
         /// <remarks>
+        /// Below is the JSON schema for the request and response payloads.
+        /// Additional information can be found in the service REST API documentation:
+        /// https://docs.microsoft.com/rest/api/AzureAIDocumentTranslation/DocumentTranslation/GetDocumentStatus
         /// Schema for <c>Response Body</c>:
         /// <code>{
-        ///   path: string,
-        ///   sourcePath: string,
-        ///   createdDateTimeUtc: string (ISO 8601 Format),
-        ///   lastActionDateTimeUtc: string (ISO 8601 Format),
-        ///   status: &quot;NotStarted&quot; | &quot;Running&quot; | &quot;Succeeded&quot; | &quot;Failed&quot; | &quot;Cancelled&quot; | &quot;Cancelling&quot; | &quot;ValidationFailed&quot;,
-        ///   to: string,
+        ///   path: string, # Optional. <Description>Location of the document or folder</Description>
+        ///   sourcePath: string, # Required. <Description>Location of the source document</Description>
+        ///   createdDateTimeUtc: string (ISO 8601 Format), # Required. <Description>Operation created date time</Description>
+        ///   lastActionDateTimeUtc: string (ISO 8601 Format), # Required. <Description>Date time in which the operation&apos;s status has been updated</Description>
+        ///   status: &quot;NotStarted&quot; | &quot;Running&quot; | &quot;Succeeded&quot; | &quot;Failed&quot; | &quot;Cancelled&quot; | &quot;Cancelling&quot; | &quot;ValidationFailed&quot;, # Required. <Description>List of possible statuses for job or document</Description>
+        ///   to: string, # Required. <Description>To language</Description>
         ///   error: {
-        ///     code: &quot;InvalidRequest&quot; | &quot;InvalidArgument&quot; | &quot;InternalServerError&quot; | &quot;ServiceUnavailable&quot; | &quot;ResourceNotFound&quot; | &quot;Unauthorized&quot; | &quot;RequestRateTooHigh&quot;,
-        ///     message: string,
-        ///     target: string,
+        ///     code: &quot;InvalidRequest&quot; | &quot;InvalidArgument&quot; | &quot;InternalServerError&quot; | &quot;ServiceUnavailable&quot; | &quot;ResourceNotFound&quot; | &quot;Unauthorized&quot; | &quot;RequestRateTooHigh&quot;, # Required. <Description>Enums containing high level error codes.</Description>
+        ///     message: string, # Required. <Description>Gets high level error message.</Description>
+        ///     target: string, # Optional. <Description>Gets the source of the error. 
+        /// For example it would be &quot;documents&quot; or &quot;document id&quot; in case of invalid document.</Description>
```

Comment:
Can we generate these docs using any tool to see how these docs will be generated? I think we can't catch all the errors without generating the docs.

Reply:
We only can generate the doc xml via `dotnet build`, but we don't know the tool to generate the msdocs. @ShivangiReja Do you know any tool ? thanks

Reply:
We can skip generating docs for this PR but it will be helpful to review the docs if we generate it for https://github.com/Azure/azure-sdk-for-net/pull/29264 PR using docs generation pipeline.